classes/class3/catHW/app.js

```diff
@@ -0,0 +1,37 @@
+var path = require('path');
+var logger = require('morgan');
+var cookieParser = require('cookie-parser');
+var bodyParser = require('body-parser');
+var express = require('express');
+var exphbs = require('express-handlebars');
+var mongoose = require('mongoose');
+
+var index = require('./routes/index');
+var schema = require('./public/schema');
+var Cat = schema.Cat;
```

Comment:
Your schema is never used in `app.js` so there's no need to be requiring it in here.